coralproject/talk
# show different copy per # of replies

## What does this PR do?

we want to show different copy depending on how many replies there are.

## How do I test this PR?

- open two windows to the same article
- make a reply on one
- go to the other and wait for the poll to update, and you should see a "View Reply" button
- if there is more than one reply in between polls
  - if it's the first time you've clicked the button, it'll say "view all _n_ replies"
  - if it's not the first time, it'll say "view _n_ replies"
